Hymns/Tags/TagStore.swift

```diff
@@ -60,6 +75,21 @@ class TagStoreRealmImpl: TagStore {
             action(favorite)
         }.toNotification()
     }
+
+    // MARK: Realm Query Functions
+
```

Comment:
What is this comment for?

Reply:
Not really needed anymore. It was more relevant when there where a million functions in tagStore